lib/ovh.js

```diff
@@ -367,7 +403,19 @@ class Ovh {
       }
     }
 
-    if (path.indexOf('/auth') < 0) {
+    if (this.oauthConfig && (!this.accessToken || this.accessToken.expired())) {
+      try {
+        this.accessToken = await this.oauthClient.getToken({scope: "all"});
+      } catch (e) {
+        return callback('[OVH] failed to retrieveOAuth2 Access Token');
```

Comment:
```suggestion
        return callback('[OVH] failed to retrieve OAuth2 Access Token');
```

Reply:
Would it be possible to extract some useful data to output?
It seems to contain something like that:

```json
  output: {
    statusCode: 401,
    payload: {
      statusCode: 401,
      error: 'Unauthorized',
      message: 'Response Error: 401 null'
    },
    headers: {}
  }